arch/arm/boot/dts/overlays/adv728x-m-overlay.dts

```diff
@@ -0,0 +1,38 @@
+// Definitions for Analog Devices ADV728[0|1|2]-M video to CSI2 bridges on VC
+// I2C bus
+/dts-v1/;
+/plugin/;
```

Comment:
Can you omit these two tags? They shouldn't really be duplicated, which is what happens when you #include.

Reply:
Done.